ui/app/utils/rollback-without-changed-attrs.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export default function rollbackWithoutChangedAttrs(model) {
```

Comment:
I spent awhile trying to figure out whether this is idiomatic or not and decided that it's actually kind of nice to take explicit control back over stuff like this.

It's annoying to write, but it's well explained and easily understood. It's a good solution!

Reply:
:) Yay